packages/components/src/menu-items-choice/stories/index.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import MenuItemsChoice from '..';
+import MenuGroup from '../../menu-group';
+
+const meta: ComponentMeta< typeof MenuItemsChoice > = {
+	component: MenuItemsChoice,
+	title: 'Components/MenuItemsChoice',
+	argTypes: {},
+	parameters: {
+		controls: {
+			expanded: true,
+		},
+		docs: { source: { state: 'open' } },
+	},
+};
+export default meta;
+
+const Template: ComponentStory< typeof MenuItemsChoice > = ( args ) => {
+	const [ choice, setChoice ] = useState( args.value ?? '' );
```

Comment:
For controlled components we usually just disable the control for `value` to avoid making things murky:

https://github.com/WordPress/gutenberg/blob/5115017b4129e942cce903a334f9d0350e21a0fe/packages/components/src/select-control/stories/index.tsx#L25

And then we can do 

```suggestion
	const [ choice, setChoice ] = useState( args.choices[ 0 ]?.value );
```

or something like that.